modules/index/src/main/java/org/apache/ignite/internal/index/SortedIndex.java

```diff
@@ -92,7 +99,9 @@ default Publisher<BinaryRow> scan(
      * @return A cursor from resulting rows.
      * @see SortedIndex#INCLUDE_LEFT
      * @see SortedIndex#INCLUDE_RIGHT
+     * @deprecated Use {@link #scan(int, UUID, ClusterNode, long, BinaryTuplePrefix, BinaryTuplePrefix, int, BitSet)} instead.
      */
+    @Deprecated
```

Comment:
please add mention IGNITE-17952  here, to be sure that we remove it

Reply:
Added jira issue number after `@deprecated` in javadoc.